conf/services/trusts.yml

```diff
@@ -0,0 +1,70 @@
+serviceName: Register and Maintain a Trust
+serviceDescription: |
+  This service allows you to register and maintain a trust which has to pay tax on the assets.
```

Comment:
Maybe we should have a _new_ description as we don't only deal with tax anymore.